src/app/clients/ClientsPage.tsx

```diff
@@ -48,24 +48,25 @@ const collectionCentreToAbbreviation = (
     }
 };
 
-export const clientTableHeaderKeysAndLabels: TableHeaders = [
-    ["iconsColumn", ""],
+export const clientTableHeaderKeysAndLabels: TableHeaders<ClientsTableRow> = [
     ["fullName", "Name"],
     ["familyCategory", "Family"],
     ["addressPostcode", "Postcode"],
     ["deliveryCollection", ""],
-    ["packingDate", "Packing Date"],
+    ["packingDatetime", "Packing Date"],
     ["packingTimeLabel", "Time"],
     ["lastStatus", "Last Status"],
+    ["voucherNumber", "Voucher"],
 ];
 
-const toggleableHeaders = [
+const toggleableHeaders: readonly (keyof ClientsTableRow)[] = [
```

Comment:
what does readonly do that isn't the same as const?

Reply:
It asserts that this is immutable and allows me to accept data declared `as const`